src/components/dashboard/components/dictionary/messages/InlineError.jsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const InlineError = ({ text }) => (
-  <div><span style={{ color: '#880000' }}> {text} </span></div>
+  <div><span id="inline-error"> {text} </span></div>
```

Comment:
Is this change part of the ticket?

Reply:
It is related to the ticket since it will help the user to clearly see the errors in the form upon submission.